android/app/src/main/java/org/fossasia/openevent/data/SessionType.java

```diff
@@ -8,6 +9,7 @@
 import io.realm.annotations.PrimaryKey;
 
 @Type("session-type")
+@JsonIgnoreProperties(ignoreUnknown = true)
 public class SessionType extends RealmObject {
```

Comment:
Even if there are no kebab attributes here, please provide the annotation for making it future safe